docs/walkthroughs/getting-started-installation.adoc

```diff
@@ -0,0 +1,62 @@
+[[getting-started]]
+= Getting started: Installation
+
+[[overview]]
+= Overview
+
+The aim of this document is to get you to the point where you have installed and enabled the mobile pieces and setup required configuration that will make mobile development against OpenShift as simple and accessible as possible. It will also explain how you can verify your OpenShift installation is ready after to start developing your mobile application against once the installation has been completed.
+
+In other guides we will show you how to make use of this installation by representing your mobile clients as resources in OpenShift, provisioning mobile enabled services such as authentication and push notification, along with consuming these services from your mobile clients via our SDKs.
+
+
+
+
+
+[[installation-options]]
+= Installation Options
```

Comment:
Should the installer from mobile-core also be added in here as an installation option?